conda_forge_tick/migrators.xsh

```diff
@@ -508,9 +508,10 @@ class Rebuild(Migrator):
                        '{{% set build = {} %}}')
                      )
 
-    def __init__(self, graph, pr_limit=0):
+    def __init__(self, graph, name, pr_limit=0):
```

Comment:
I think it you are going to change the API for one migrator in a required way, you should change it for all migrators.  Like perhaps this should be on the `Migrator` class or a keyword argument.

Reply:
:+1: for having name as kwarg